robyn/import_vars.py

```diff
@@ -0,0 +1,27 @@
+import os 
+from definitions import CONFIG_PATH
+
+# parse the configuration file returning a list of tuples (key, value) containing the environment variables
+def parser():
+    """Parse the configuration file"""
+    with open(CONFIG_PATH, 'r') as f:
+        for line in f:
+            if line.startswith('#'):
+                continue
+            yield line.strip().split('=')  
+
```

Comment:
I would recommend writing the function as the following

```python

def parser(config_path=CONFIG_PATH):
    ...

```

Declaring a function like this will allow it to be more testable and not have us mock the variables if we need to write any unit tests.

This is called reducing coupling. You can have a look at this concept here: https://www.youtube.com/watch?v=eiDyK_ofPPM